src/main/java/org/waarp/openr66/protocol/localhandler/LocalTransaction.java

```diff
@@ -314,20 +313,6 @@ public LocalChannelReference createNewClient(NetworkChannelReference networkChan
      * @return the LocalChannelReference
      */
     public LocalChannelReference getFromId(Integer id) {
-        int maxtry = (int) (Configuration.configuration.getTIMEOUTCON() / Configuration.RETRYINMS) / 2;
-        maxtry = 100;
-        for (int i = 0; i < maxtry; i++) {
-            LocalChannelReference lcr = localChannelHashMap.get(id);
-            if (lcr == null) {
-                try {
-                    Thread.sleep(Configuration.RETRYINMS);
-                    Thread.yield();
-                } catch (InterruptedException e) {
-                }
-            } else {
-                return lcr;
-            }
-        }
```

Comment:
Warning: I will propose probably in 2 months (delay) to correct the current way the network and localchannel are bind together. Until then, the localChannelHashMap could be not ready (not having the id item) as soon as this method is called... So the try and wait... You should keep it for the moment...

Reply:
Not needed anymore